src/libraries/System.Diagnostics.Process/tests/ProcessModuleTests.cs

```diff
@@ -76,7 +77,15 @@ public void ModulesAreDisposedWhenProcessIsDisposed()
         {
             Process process = CreateDefaultProcess();
 
-            ProcessModuleCollection modulesCollection = process.Modules;
+            // Very rarely, this call will fail with ERROR_PARTIAL_COPY; it only happened
+            // so far on this particular test, the only one that creates a new process.
+            // Assumption is that we need to give a little extra time.
+            ProcessModuleCollection modulesCollection = null;
+            RetryHelper.Execute(() =>
+            {
+                modulesCollection = process.Modules;
+            }, maxAttempts:5, backoffFunc:null, retryWhen:e => { return (e.GetType() == typeof(Win32Exception)); });
```

Comment:
minor (and opinionated nit):

```suggestion
            }, maxAttempts: 5, backoffFunc: null, retryWhen: e => e.GetType() == typeof(Win32Exception));
```

Reply:
51 matches for `, \w+:\w+,` and 2426 for `, \w+: \w+,` -- you win :) @sharwell is there a rule we can enable to force a space in these?